backend/tests/test_integration_org_unit.py

```diff
@@ -918,8 +920,14 @@ def test_edit_org_unit(self):
 
         self.assertRegistrationsEqual(expected, actual)
 
+    @unittest.expectedFailure
```

Comment:
Not a fan of this. Can we edit the tests to still pass? History shows that once a test is marked `skip` or `expectedFailure`, no-one will care or notice that it's wrong 😕

Reply:
Removed 2 expected failures